kano_avatar/logic.py

```diff
@@ -668,6 +674,8 @@ def get_avatar_conf(aux_files=[]):
         content_dir.register_path(
```

Comment:
We will need to change the spec name once the new content object spec is created.